### PR TITLE
Add Onboarding to the app

### PR DESCRIPTION
**Pull request by Vanethos**
Add onboarding to the app
Add request of notification permissions for iOS
Add VOST logo as an asset

NOTE: The texts are still as Strings since we don't have the translations to english

### CODE REVIEW

File: lib/presentation/ui/_base/base_page.dart [hunk 1]
```diff
@@ -23,8 +23,10 @@ abstract class BaseState<T extends BasePage> extends State<T> {
   @override
   @mustCallSuper
   void initState() {
+    super.initState();
     widget.bloc.errorStream
-        .transform(new ThrottleStreamTransformer(new Duration(seconds: 2)))
+        .transform(new ThrottleStreamTransformer(
```
Comment (miguelpruivo) [inline lib/presentation/ui/_base/base_page.dart:28]: What are you actually trying to achieve here?

File: lib/presentation/ui/intro/intro_page.dart [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+
+class _IntroState extends State<IntroPage> {
+  PageController _pageController;
+  GlobalKey<ScaffoldState> key;
+
+  _IntroState();
+
+  int sliderAnimationSpeed = 300;
+
+  var _numberOfPages = 3;
```
Comment (miguelpruivo) [inline lib/presentation/ui/intro/intro_page.dart]: I’d use explicit types here.

**Review by miguelpruivo — DISMISSED** [review]
@Vanethos LGMT, nevertheless, check the comments. 👍🏻

File: lib/presentation/ui/_base/base_page.dart (hunk 1 above)
Comment (Vanethos) [reply lib/presentation/ui/_base/base_page.dart:28]: That is from an old commit ahaha

Well, if an error is fired twice in a rapid succession (2 API calls fail at the same time, they will add an event to the error stream), the Snackbar will be shown multiple times.

If we do it like this then we are throttling the events with the approximate duration of a Snackbar LONG

**Comment by Vanethos** [comment]
Pushed changes

**Review by miguelpruivo — COMMENTED** [review]
LGTM

**Comment by Vanethos** [comment]
@miguelpruivo if you don't mind since you have the OK I'll merge this!

**Pull request merged**